JustSaying/project.json

```diff
@@ -37,10 +37,10 @@
     "AWSSDK.SimpleNotificationService": "3.3.0.6",
     "AWSSDK.SQS": "3.3.1.6",
     "JustSaying.AwsTools": "5.0.0",
+    "JustSaying.Logging": "5.0.0",
```

Comment:
Yeah, this is broken, but you are right the build is broken atm anyway. Good spot